Pinta.Core/Classes/Document.cs

```diff
@@ -421,7 +421,9 @@ public void FinishSelection ()
 			Layer layer = SelectionLayer;
 
 			using (Cairo.Context g = new Cairo.Context (CurrentUserLayer.Surface)) {
-				layer.Draw(g);
+				//layer.Draw(g);
```

Comment:
Can you remove this line instead of leaving it commented out? Otherwise, the changes look good